plexpy/datafactory.py

```diff
@@ -1156,6 +1156,18 @@ def get_watch_time_stats(self, rating_key=None, grouping=None, query_days=None):
 
         group_by = 'session_history.reference_id' if grouping else 'session_history.id'
 
+        rating_keys = []
+        if media_type == 'collection':
+            pms_connect = pmsconnect.PmsConnect()
+            result = pms_connect.get_item_children(rating_key=rating_key)
```

Comment:
This works with artist/album/season/episode collections by passing `media_type`.
```suggestion
            result = pms_connect.get_item_children(rating_key=rating_key, media_type=media_type)
```

Reply:
Good to know.